### PR TITLE
feat(react-redux-spark-metrics): Add Metrics module

### PR DESCRIPTION
**Pull request by bzang**
Not ready for primetime yet

### CODE REVIEW

**Comment by adamweeks** [comment]
@bzang let me know when this is ready, looks like it is passing CI now

**Comment by bzang** [comment]
@adamweeks, i think we're good to go with the first version of this

File: packages/node_modules/@ciscospark/react-redux-spark-metrics/src/reducer.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+  case STORE_METRIC:
+    return state.setIn([`items`, action.payload.name], fromJS(action.payload.data));
+  case DELETE_METRIC:
+    return state.deleteIn([`items`, action.payload.name]);
+  case ADD_TO_QUEUE: {
+    const newQueue = state.get(`queue`).push(fromJS(action.payload.data));
+    return state.set(`queue`, newQueue);
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-redux-spark-metrics/src/reducer.js]: I'm surprised there's no `pushIn` that we could use.

File: packages/node_modules/@ciscospark/widget-space-demo/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@ciscospark/widget-space",
+  "name": "@ciscospark/widget-space-demo",
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-space-demo/package.json]: Good catch.

**Pull request merged**